.github/workflows/Dockerfile.ci

```diff
@@ -0,0 +1,68 @@
+FROM debian:bullseye AS lldap
+ARG DEBIAN_FRONTEND=noninteractive
+ARG TARGETPLATFORM
+RUN apt update && apt install -y wget
+WORKDIR /dim
+COPY bin/ bin/
+COPY web/ web/
+
+RUN mkdir -p target/
+RUN mkdir -p /lldap/app
+
```

Comment:
Aren't you missing a build step here?

Reply:
it for preparing copy file, as i COPY all the binary and the static first into container.
the web -> /lldap/app
the bin/ -> target/ 
and both copy out the last stage.